pkg/binding/list.go

```diff
@@ -0,0 +1,76 @@
+package binding
+
+import (
+	"github.com/devfile/library/pkg/devfile/parser"
+	"github.com/redhat-developer/odo/pkg/api"
+	"github.com/redhat-developer/odo/pkg/labels"
+)
+
+func (o *BindingClient) ListAllBindings(devfileObj parser.DevfileObj, context string) ([]api.ServiceBinding, []string, error) {
```

Comment:
I think unit-testing this method, if possible, could be helpful. WDYT?

Reply:
Yes, you are right, I will see how to add unit tests for this function